### PR TITLE
Security scans (JAVA)

### PR DESCRIPTION
**Pull request by leobr29**
Implement automatic security scan for java with Trivy

### CODE REVIEW

**Comment by leobr29** [comment]
ClickUp : https://app.clickup.com/t/8696khz8y

File: java-demo/README.md
Comment (roje-bodc) [inline java-demo/README.md:133]: typo here with 'teh'.

**Comment by roje-bodc** [comment]
Happy for this to be merged following action on the minor comments I have made 👍

**Review by roje-bodc — DISMISSED** [review]

**Comment by leobr29** [comment]
Hi @roje-bodc, seems that it require another review after merging the other pull request

**Review by roje-bodc — DISMISSED** [review]

**Review by roje-bodc — DISMISSED** [review]

**Review by roje-bodc — DISMISSED** [review]

**Comment by roje-bodc** [comment]
Not sure why my reviews are being dismissed 🤔

**Comment by leobr29** [comment]
@roje-bodc don't understand why... try to review a last time see if it works if I validate just after ?!

**Review by roje-bodc — DISMISSED** [review]

**Comment by leobr29** [comment]
It seem's that we need a review grom @MCazaly too ?!

image

**Review by roje-bodc — DISMISSED** [review]

**Comment by roje-bodc** [comment]
> It seem's that we need a review from @MCazaly too ?!
> 
> image

Hmmm potentially? He's finished for the day now though. We can try to ask him to approve tomorrow if he's available 😅

**Comment by leobr29** [comment]
@roje-bodc I just created another pull request for this dev https://github.com/British-Oceanographic-Data-Centre/amrit-repos/pull/15

**Comment by roje-bodc** [comment]
> @roje-bodc I just created another pull request for this dev #15

@leobr29 Seems to have worked!

**Pull request merged**
